src/placement/jump_map.c

```diff
@@ -1068,7 +1071,7 @@ jump_map_obj_place(struct pl_map *map, struct daos_obj_md *md,
 
 	D_INIT_LIST_HEAD(&extend_list);
 	allow_status = PO_COMP_ST_UPIN | PO_COMP_ST_DRAIN;
-	rc = obj_layout_alloc_and_get(jmap, &jmop, md, allow_status, &layout,
+	rc = obj_layout_alloc_and_get(jmap, &jmop, md, allow_status, -1 ,&layout,
```

Comment:
(style)  space prohibited before that ',' (ctx:WxO)
(style)  space required after that ',' (ctx:WxO)
(style)  space required before that '&' (ctx:OxV)